qiskit/circuit/library/__init__.py

```diff
@@ -306,9 +306,9 @@
 N-local circuits
 ================
 
-These :class:`~qiskit.circuit.library.BlueprintCircuit`
-subclasses are used as initial guesses (a.k.a. ansatzes or variational forms) in chemistry.
-
+These :class:`~qiskit.circuit.library.BlueprintCircuit` subclasses are used 
+as parameterized models (a.k.a. ansatzes or variational forms) in variational algorithms.
+They are heavily used near-term algorithms in e.g. Chemistry, Physics or Optimization.
```

Comment:
Whoops my suggestion had a typo, sorry!
```suggestion
They are heavily used in near-term algorithms in e.g. Chemistry, Physics or Optimization.
```